tutorials/selenium/target-app/build.gradle.kts

```diff
@@ -18,12 +18,33 @@
  *  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-plugins {
-    id("edu.sc.seis.launch4j") version "3.0.4"
+val mainJar = "targetApp.jar"
+
+tasks.jar {
+    archiveFileName.set(mainJar)
+    manifest {
+        attributes["Main-Class"] = "TargetApp"
```

Comment:
Why `TargetApp` and not just `App` or `Demo`?

Reply:
good point